### PR TITLE
Make theme compatible with Plone 5.1.

### PR DESCRIPTION
**Pull request by mbaechtold**
Close https://github.com/4teamwork/izug.organisation/issues/1458

In this commit we drop plone 4 support and add plone 5 support.

Since a lot of code changed we decided that we will not support both anymore. The 3.x branch will be continued for plone4 projects.

### CODE REVIEW

File: plonetheme/onegov/utils.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from plone import api
 from Products.CMFCore.utils import getToolByName
+import pkg_resources
```
[the remark this answers is not in the document]
Comment (mbaechtold) [reply plonetheme/onegov/utils.py]: My bad. Needs to be removed.

**Comment by busykoala** [comment]
Fix search box styles.

image

image

**Comment by busykoala** [comment]
Fix padding in controlpanel.

image

**Comment by busykoala** [comment]
Style action control panel buttons.

image

File: plonetheme/onegov/resources/js/onegov-compiled.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    if (c==0) setTimeout("c=0;", 2000);
+    c += 1;
+    if (c==10) $('#custom-scss-field').fadeIn('slow');
+  });
+
+});
+
+});
+
+define("/Users/lukas/projects/plonetheme.onegov/plonetheme/onegov/resources/js/onegov.js", function(){});
```
[the remark this answers is not in the document]
Comment (busykoala) [reply plonetheme/onegov/resources/js/onegov-compiled.js]: I remove it in 9294f6e

File: plonetheme/onegov/resources/js/onegov-compiled.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
+requirejs([
```
[the remark this answers is not in the document]
Comment (busykoala) [reply plonetheme/onegov/resources/js/onegov-compiled.js]: I removed the allegedly compiled one and replaced the name in the min.js file with the uncompiled file.

File: plonetheme/onegov/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (busykoala) [reply plonetheme/onegov/utils.py]: Is removed in 3882758.

File: plonetheme/onegov/viewlets/configure.zcml [hunk 4]
```diff
@@ -24,6 +25,15 @@
 
   <!-- The global sections -->
   <browser:viewlet
+      zcml:condition="have plone-5"
```
[the remark this answers is not in the document]
Comment (busykoala) [reply plonetheme/onegov/viewlets/configure.zcml]: It is fixed in a475063. We hadn't decided that there is a branch for each version of plone by the time. But you are right of course!

**Comment by busykoala** [comment]
The contents bar had a higher z-index than the menu. The commit 11604e2 moves it to the back a bit.

image

image

File: plonetheme/onegov/profiles/default/skins.xml [hunk 5]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
```
[the remark this answers is not in the document]
Comment (busykoala) [reply plonetheme/onegov/profiles/default/skins.xml]: They are removed in f376988.

[the remark this answers is not in the document]
Comment (djowett-ftw) [reply plonetheme/onegov/profiles/default/skins.xml]: 👍

File: plonetheme/onegov/resources/js/onegov-compiled.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (djowett-ftw) [reply plonetheme/onegov/resources/js/onegov-compiled.js]: 👍

File: plonetheme/onegov/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (djowett-ftw) [reply plonetheme/onegov/utils.py]: 👍

File: plonetheme/onegov/viewlets/configure.zcml (hunk 4 above)
[the remark this answers is not in the document]
Comment (djowett-ftw) [reply plonetheme/onegov/viewlets/configure.zcml]: 👍

**Comment by Nachtalb** [comment]
@jone as discussed earlier I have added support for both the new plone 5 `globalstatusmessage` DOM as well as `ftw.globalstatusmessage`'s integration. One problem remains though. I cannot push to this repo 😂 because I am not part of @OneGov. How do I become a member?

**Comment by jone** [comment]
> @jone as discussed earlier I have added support for both the new plone 5 `globalstatusmessage` DOM as well as `ftw.globalstatusmessage`'s integration. One problem remains though. I cannot push to this repo 😂 because I am not part of @OneGov. How do I become a member?

I have sent you another invitation (there was one pending).

But I think there is a missunderstandig:
I was expecting that we change the markup in `ftw.globalstatusmessage`.
`izug.organisation` and `plonetheme.onegov` will no longer be Plone 4 compatible.

**Pull request merged**